spec/borda_poll_spec.rb

```diff
@@ -0,0 +1,90 @@
+require_relative '../lib/borda_poll'
+
+expired_date = Date.new(1970, 1, 1)
+valid_date = Date.today + 2
+
+RSpec.describe BordaPoll do
```

Comment:
RSpec では複数のテストで共通して使用する変数を `let` ヘルパーを使って定義するのが一般的です。
https://relishapp.com/rspec/rspec-core/v/3-10/docs/helper-methods/let-and-let
（`app_spec.rb` でも `let` を使用しています。）

```suggestion
RSpec.describe BordaPoll do
  let(:expired_date) { Date.new(1970, 1, 1) }
  let(:valid_date) { Date.today + 2 }
```

[imo] ただ、どちらも1つのテストでしか使われてないので、それぞれのテスト内でローカル変数にしても良いかなと思いました。

Reply:
あら、確かに `app_spec.rb` でも `let` が使われていますね（気づかず）
ありがとうございます